src/patterns/technical-problem-pages/index.md.njk

```diff
@@ -0,0 +1,85 @@
+---
+title: Technical problem pages
+description: This is a page that tells someone there is something wrong with the service. They are also known as 500 pages.
+section: Patterns
+theme: Pages
+aliases:
+backlog_issue_id: 129
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This pattern is currently experimental because <a class="govuk-link" href="#research-on-this-pattern">more research</a> is needed to validate it.
+---
+{% from "_example.njk" import example %}
+
+This is a page that tells someone there is something wrong with the service. They are also known as 500 and internal server error pages.
+
+{{ example({group: 'patterns', item: 'technical-problem-pages', example: 'default', html: true, open: false}) }}
+
+## When to use this pattern
+
+Use this page when there is an unexpected problem with the service. Use the same page for all unexpected problems.
+
+Log all errors and fix them as quickly as possible.
+
+Only display the page for a short time. If a problem cannot be fixed quickly, close the service and use a service unavailable page.
```

Comment:
Link to service unavailable pages.

Reply:
Will be done after these patterns are both live, since they exist separately.